src/codegen/expression.rs

```diff
@@ -812,7 +812,13 @@ pub fn expression(
         } => {
             let expr = expression(&args[0], cfg, contract_no, func, ns, vartab, opt);
 
-            cfg.add(vartab, Instr::Print { expr });
+            let to_print = if ns.target.is_substrate() {
+                add_prefix_and_dilimiter_to_print(expr)
```

Comment:
It is called `delimiter`

```suggestion
                add_prefix_and_delimiter_to_print(expr)
```